Library/Homebrew/keg_relocate.rb

```diff
@@ -87,15 +87,21 @@ def change_rpath(file, new_prefix)
     end
     return unless patchelf.installed?
     cmd = "#{patchelf.bin}/patchelf --set-rpath #{new_prefix}/lib"
+    (old_i, old_p) = `#{patchelf.bin}/patchelf --print-rpath --print-interpreter #{file}`.split("\n")
```

Comment:
This command fails when the file is a dynamic library rather than an executable.
```
$ patchelf --print-rpath --print-interpreter /home/sjackman/.linuxbrew/Cellar/zlib/1.2.8/lib/libz.so.1.2.8
cannot find section .interp
```
Reading `--print-interpreter` will have to be moved down below inside `if file.mach_o_executable?`